src/Data/Formatter/Number.purs

```diff
@@ -118,6 +120,11 @@ format (Formatter f) num =
            let
              multiplier = Math.pow 10.0 $ Int.toNumber f.after
            in Int.round $ leftover * multiplier
+         roundedWithZeros =

```

Comment:
I don't understand this part, @cryogenian can you take a look?

Reply:
This adds zeros after comma/point and digits.